index.html

```diff
@@ -65,6 +66,356 @@ <h3>I'm a Software Developer</h3>
+        </article>
+        <article class="work-card">
+          <div class="work-contents">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>
+                <a
+                  href="https://developer.mozilla.org/en-US/docs/Web/HTML"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  HTML
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Bootstrap
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/documentation/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  Ruby
+                </a>
+              </li>
+            </ul>
+          </div>
+
+          <button type="button"><a href="#"> See project </a></button>
+        </article>
+      </section>
+
+      <!-- About-->
+      <section class="about-container">
+        <article class="about-hero">
+          <h2>About me</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <button type="button" class="about-btn">
+            <a href="#" target="_blank"> Get my resume </a>
+          </button>
+        </article>
+        <article class="about-skills">
+          <ul class="skill-sets-wrapper">
+            <div class="skill-title">
+              <div class="icon" id="language"></div>
+              <h3>Language</h3>
+            </div>
+            <li class="skill-sets">JavaScript</li>
+            <li class="skill-sets">Ruby</li>
+            <li class="skill-sets">HTML</li>
+            <li class="skill-sets">CSS</li>
+          </ul>
+          <ul class="skill-sets-wrapper">
+            <div class="skill-title">
+              <div class="icon" id="frameworks"></div>
+              <h3>Frameworks</h3>
+            </div>
+            <li class="skill-sets">React.js</li>
+            <li class="skill-sets">Ruby on Rails</li>
+            <li class="skill-sets">RSpec</li>
+            <li class="skill-sets">Capybara</li>
+          </ul>
+          <ul class="skill-sets-wrapper">
+            <div class="skill-title">
+              <div class="icon" id="skills"></div>
+              <h3>Skills</h3>
+            </div>
+            <li class="skill-sets">Database Management</li>
```

Comment:
- [ ] You really did a great job here :clap: However it would be nice to wrap the technology categories ("Languages", "Frameworks" and "Skills") in a `ul` and each of them in a `li` for semantics reason. Because it is a list of categories. 
E.g.
```html
<ul>
  <li>
    <h3>Languages</h3>
    <ul>
      <li>html</li>
      ...
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
    <ul>
      <li>React</li>
      ...
    </ul>
  </li>
  ...
</ul>